lint.swift

```diff
@@ -0,0 +1,547 @@
+        nonMatchingExamples: ["- Fixed a bug.\n  Issue: [Link](#)"],
+        includeFilters: [changelogFile],
+        autoCorrectReplacement: "\n  $2",
+        autoCorrectExamples: [
+            ["before": "\n- Fixed a bug.\nIssue: [Link](#)", "after": "\n- Fixed a bug.\n  Issue: [Link](#)"],
+            ["before": "\n- Fixed a bug.\n Issue: [Link](#)", "after": "\n- Fixed a bug.\n  Issue: [Link](#)"],
+            ["before": "\n- Fixed a bug.\n    Issue: [Link](#)", "after": "\n- Fixed a bug.\n  Issue: [Link](#)"],
+        ]
+    )
+
+    // MARK: ClosureParamsParantheses
+    try Lint.checkFileContents(
+        checkInfo: "ClosureParamsParantheses: Don't use parantheses around non-typed parameters in a closure.",
+        regex: #"\{\s*\(((?!self)[^):]+)\)\s*in"#,
+        matchingExamples: ["run { (a) in", "run { (a, b) in", "run { (a, b, c) in"],
+        nonMatchingExamples: ["run { (a: Int) in", "run { (a: Int, b: Int) in", "run { (a, b) -> String in"],
+        includeFilters: [swiftFiles],
+        excludeFilters: [testFiles],
+        autoCorrectReplacement: #"{ $1 in"#,
+        autoCorrectExamples: [
+            ["before": "run { (a) in", "after": "run { a in"],
+            ["before": "run { (a, b) in", "after": "run { a, b in"],
+            ["before": "run { (a, b, c) in", "after": "run { a, b, c in"]
+        ]
+    )
+
+    // MARK: EmptyMethodBody
+    try Lint.checkFileContents(
+        checkInfo: "EmptyMethodBody: Don't use whitespace or newlines for the body of empty methods.",
+        regex: ["declaration": #"(init|func [^\(\s]+)\([^{}]*\)"#, "spacing": #"\s*"#, "body": #"\{\s+\}"#],
+        matchingExamples: [
+            "init() { }",
+            "init() {\n\n}",
+            "init(\n    x: Int,\n    y: Int\n) { }",
+            "func foo2bar()  { }",
+            "func foo2bar(x: Int, y: Int)  { }",
+            "func foo2bar(\n    x: Int,\n    y: Int\n) {\n    \n}"
+        ],
+        nonMatchingExamples: ["init() { /* comment */ }", "init() {}", "func foo2bar() {}", "func foo2bar(x: Int, y: Int) {}"],
+        includeFilters: [swiftFiles],
+        excludeFilters: [testFiles],
+        autoCorrectReplacement: "$declaration {}",
+        autoCorrectExamples: [
+            ["before": "init()  { }", "after": "init() {}"],
+            ["before": "init(x: Int, y: Int)  { }", "after": "init(x: Int, y: Int) {}"],
+            ["before": "init()\n{\n    \n}", "after": "init() {}"],
+            ["before": "init(\n    x: Int,\n    y: Int\n) {\n    \n}", "after": "init(\n    x: Int,\n    y: Int\n) {}"],
+            ["before": "func foo2bar()  { }", "after": "func foo2bar() {}"],
+            ["before": "func foo2bar(x: Int, y: Int)  { }", "after": "func foo2bar(x: Int, y: Int) {}"],
+            ["before": "func foo2bar()\n{\n    \n}", "after": "func foo2bar() {}"],
+            ["before": "func foo2bar(\n    x: Int,\n    y: Int\n) {\n    \n}", "after": "func foo2bar(\n    x: Int,\n    y: Int\n) {}"]
```

Comment:
Codacy found an issue: [Multi-line collection literals should have trailing commas.](https://app.codacy.com/gh/Flinesoft/HandySwift/pullRequest?prid=5730214)